kuma/users/jinja2/users/ban_user.html

```diff
@@ -0,0 +1,53 @@
+{% extends "base.html" %}
+
+{% block body_attributes %}{% endblock %}
+
+{% block bodyclass %}user ban-user{% endblock %}
+
+{% block title %}Ban {{ user_to_ban }}{% endblock %}
+
+{% block site_css %}
+  {{ super() }}
+  {% stylesheet 'users' %}
+{% endblock %}
+
+{% block content %}
+<section id="content">
+<div class="wrap">
+  <section id="content-main" class="full">
+    <section id="user-head" class="column-container column-container-reverse vcard user-head">
+      <div class="column-10">
+        <h1 class="user-title">
+          {{ _('Why are you banning') }} {{user_to_ban }}?
+        </h1>
+        <div class="column-4">
+          <h5>{{ _('Choose a reason') }}:</h5>
+          {% for reason in common_reasons %}
+            <div><a class="ban-common-reason">{{ _(reason) }}</a></div>
```

Comment:
Since this isn't a link that takes the user off the page an `a` element is not appropriate (and the browser responds to the missing href by not changing the cursor when the element is hovered). I'm not familiar with the purpose of the PR but since you are looping through a list of options for the user to pick between I would suggest either `select` or `button`.